Support/Multitenancy/Ironic_bmo-mult-instances/dowanload-node-image.sh

```diff
@@ -1,5 +1,5 @@
-cd /opt/metal3-dev-env/ironic/html/images/
+cd /opt/metal3-dev-env/ironic/html/images/ || exit
 wget https://artifactory.nordix.org/artifactory/metal3/images/k8s_v1.24.1/CENTOS_9_NODE_IMAGE_K8S_v1.24.1.qcow2
 qemu-img convert -O raw CENTOS_9_NODE_IMAGE_K8S_v1.24.1.qcow2 CENTOS_9_NODE_IMAGE_K8S_v1.24.1-raw.img
 md5sum CENTOS_9_NODE_IMAGE_K8S_v1.24.1-raw.img | awk '{print $1}' > CENTOS_9_NODE_IMAGE_K8S_v1.24.1-raw.img.md5sum
-cd -
+cd - || exit
```

Comment:
If this script isn't sourced to another script, then you might delete this line as previous current directory is automatically restored. If it is sourced, then see above.

Reply:
not sure I will keep it as it is